libraries/postgres/test/database_test.js

```diff
@@ -820,8 +820,8 @@ helper.dbSuite(path.basename(__filename), function() {
         async () => {
           await db.fns.add_from_numbers({ a: 1, b: 2 });
         },
-        /function add_from_numbers(unknown) does not exist/
-      )
+        UNDEFINED_FUNCTION,
```

Comment:
I thought this needed to be a regex if it was trying to find a string

Reply:
You're absolutely right. This was a false positive. Fixed!